pkg/time/time.go

```diff
@@ -21,6 +21,15 @@ import (
 )
 
 const (
+	// YearMonthDay is a time format similar to RFC3339 with day granularity
+	// instead of second.
+	YearMonthDay = "2006-01-02"
+	// YearMonthDayHour is a time format similar to RFC3339 with hour
+	// granularity instead of second.
+	YearMonthDayHour = "2006-01-02T15Z"
```

Comment:
If the goal of the trailing Z is to allow specifiying the tz, it should be:
```suggestion
	YearMonthDayHour = "2006-01-02T15Z07:00"
```